process/project_proposals.adoc

```diff
@@ -22,5 +22,7 @@
  .. project logo in svg format (see https://github.com/cncf/artwork#cncf-related-logos-and-artwork for guidelines)
 
 . *Project Acceptance Process*.
+ .. Project is proposed via a GitHub issue
+ .. TOC may request that a [CNCF SIG](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md) assesses the project 
```

Comment:
@lizrice this is not a markdown file. so the link format does not apply. we need to change it to link[content]. see line 18 for example.

Reply:
thanks @xiang90, fixed